apps/block_scout_web/assets/css/app.scss

```diff
@@ -81,7 +81,7 @@ $fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
 @import "components/card";
 @import "components/tile";
 @import "components/dashboard-banner";
-@import "components/datastudio";
+@import "components/stats";
```

Comment:
Since you replaced `components/datastudio` with `components/stats`, the original file can be removed.